hardhat.config.js

```diff
@@ -1,20 +1,20 @@
-usePlugin('@nomiclabs/buidler-truffle5');
-usePlugin('@nomiclabs/buidler-web3');
-usePlugin('solidity-coverage');
+require("@nomiclabs/hardhat-truffle5");
+require("@nomiclabs/hardhat-web3");
 
 require('dotenv').config();
 require('./deployment/multipleRewards');
```

Comment:
I think can remove this one, was using it for deployment last time....

Reply:
Which line? The multipleRewards one?